ts/input/tex/color/ColorMethods.ts

```diff
@@ -140,15 +139,16 @@ ColorMethods.ColorBox = function (parser: TexParser, name: string) {
  * @param {string} name The name of the control sequence.
  */
 ColorMethods.FColorBox = function (parser: TexParser, name: string) {
+  const model = parser.GetBrackets(name, "");
   const fname = parser.GetArgument(name);
   const cname = parser.GetArgument(name);
```

Comment:
I think the way to do this is
```suggestion
  const fmodel = parser.GetBrackets(name, 'named');
  const fname = parser.GetArgument(name);
  const cmodel = parser.GetBrackets(name, fmodel);
  const cname = parser.GetArgument(name);
```

and then use `fmodel` to get the color model for the frame and `cmodel` to get the model for the background color.  So you will need `frameModel` in addition to `colorModel` below.  (I haven't tested it, but this should be the right idea.)

Reply:
Thanks! I'll try to find time this week.